scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1029,12 +1029,65 @@ public void changeJobPriority(JobId jobId, JobPriority priority)
         schedulingService.changeJobPriority(jobId, priority);
     }
 
+    /**
+     * This method blocks until job is actually finished its submission.
+     * @param jobId
+     */
+    private void blockUntilJobIsUnderSubmission(JobId jobId) {
```

Comment:
This method name is not fully correct to its content.
To me, according to its implementation, we could name it 'blockUntilJobIsNotWaitingForSubmission'
(for example, think about the moment when the job is already running)